libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestScript.cs

```diff
@@ -94,6 +95,15 @@ public TestScript()
         [JsonProperty("httpRequestMocks")]
         public List<HttpRequestMock> HttpRequestMocks { get; set; } = new List<HttpRequestMock>();
 
+        /// <summary>
+        /// Gets or sets the mock data for mocking skill.
+        /// </summary>
+        /// <value>
+        /// The mock data for mocking skill.
+        /// </value>
+        [JsonProperty("skillMock")]
+        public SkillMock SkillMock { get; set; }
```

Comment:
>SkillMock [](start = 15, length = 9)

What if you need to mock multiple skills?
Should this be a List<SkillMock> ?

Reply:
removed